sig-multicluster/annual-report-2021.md

```diff
@@ -3,135 +3,82 @@
 ## Current initiatives
 
 1. What work did the SIG do this year that should be highlighted?
-
-   -
-   -
-   -
+   - Multicluster Services
+   - ClusterProperty
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
-
-   -
-   -
-   -
+   - Multicluster leader election has been discussed several times but is not currently associated with any KEP
+   - Multicluster network policy (driven by SIG-Net)
+   - Multi-network as potential MCS extension
 
 3. KEP work in 2021 (1.x, 1.y, 1.z):
 
-<!--
-In future, this will be generated from kubernetes/enhancements kep.yaml files
-1. with SIG as owning-sig or in participating-sigs
-2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
--->
-
-   - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-   - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - [1645 - Multicluster Services API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api) - Beta promotion pending Cluster ID beta
+     - [2149 - Cluster ID](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid) - Beta promotion imminent
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
-
-   -
-   -
-   -
+   
+   - Work API: progress stalled out, we should make progress or consider archival
+   - Kubefed: no progress made in the last year, considering archival
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Zoom attendance
+   - Youtube meetup views
+   - Complaints about late youtube uploads of meetups :)
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - We welcome all to our meetups to present ideas, ask questions. We're actively looking for help on a few initiatives
```

Comment:
Do you happen to have any issues or docs that could be linked here? When we do our large report out, anything that we can use to direct folk is super useful 🙏